mdsf.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://raw.githubusercontent.com/hougesen/mdsf/HEAD/schemas/v0.9.1/mdsf.schema.json",
+  "format_finished_document": false,
+  "languages": {
+    "go": [
+      [
+        "gofmt",
+        "goimports"
+      ]
```

Comment:
You can even use golangci-lint

```suggestion
    "go": [
      [
        "golangci-lint:fmt",
        "gofmt",
        "goimports"
      ]
```

Reply:
But there is no golangci-lint configuration file now. It would still work

Reply:
Integration with golangci-lint will be done in #1250

Reply:
I added a [comment](https://github.com/mgechev/revive/issues/1250#issuecomment-2830881616) to the issue about the possible mdsf integration